nextjs-app/components/pages/main/WhatWeBring.tsx

```diff
@@ -0,0 +1,87 @@
+"use client";
+import Button from "@/components/common/Button";
+import Image from "next/image";
+import { useRouter } from "next/navigation";
+
+interface GoalCardProps {
```

Comment:
Can we use I${component}Props for the type definition of component properties ?
cc @ChengYiLin 
```suggestion
interface IGoalCardProps {
```

Reply:
I agree

Reply:
Noted. Is this part of a naming convention?

Reply:
Yes, I think so.
I also followed this convention in my previous company, adding `I` prefix to Interfaces.